### ISSUE TITLE
Can't parse fragments

### ISSUE THREAD

**Issue by earboxer** [TechWilk/bible-verse-parser#62]
Can't parse verses with a frament in them: e.g.

```
Philippians 2:14-15a
Matthew 5:14, 16a
Matthew 6:25, 26 &34a
```

**Comment by TechWilk** [comment]
Thanks very much for identifying and working on both this and #63!
I was aware of both of the issues but hadn't got round to solving at them yet so it's a great help.

I'll try and find time this week to go through the PRs in detail to get them merged

**Comment by TechWilk** [comment]
Merged in #77
